app/scripts/components/common/page-header/nav-menu-item.tsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+import styled from 'styled-components';
+import { NavLink } from 'react-router-dom';
+import {
+  glsp,
+  media,
+  rgba,
+  themeVal
+} from '@devseed-ui/theme-provider';
+import { Button } from '@devseed-ui/button';
+import { DropMenu, DropMenuItem } from '@devseed-ui/dropdown';
+
+import DropdownScrollable from '../dropdown-scrollable';
+import GoogleForm from '../google-form';
+import { AlignmentEnum, InternalNavLink, ExternalNavLink, NavLinkItem, DropdownNavLink, ModalNavLink, NavItem } from './types';
+
+import GlobalMenuLinkCSS from '$styles/menu-link';
+import { useMediaQuery } from '$utils/use-media-query';
+
+const rgbaFixed = rgba as any;
+
+export const GlobalNavActions = styled.div`
+  align-self: start;
+  ${media.largeUp`
+    display: none;
+  `}
+`;
+
+const GlobalMenuItem = styled.span`
+  ${GlobalMenuLinkCSS}
+  cursor: default;
+  &:hover {
+    opacity: 1;
+  }
+`;
+
+export const GlobalNavToggle = styled(Button)`
+  z-index: 2000;
+`;
+
+const GlobalMenuLink = styled(NavLink)`
+  ${GlobalMenuLinkCSS}
+`;
+const GlobalMenuButton = styled(Button)`
+  ${GlobalMenuLinkCSS}
+`;
+
+const DropMenuNavItem = styled(DropMenuItem)`
+  &.active {
+    background-color: ${rgbaFixed(themeVal('color.link'), 0.08)};
+  }
+  ${media.largeDown`
+    padding-left ${glsp(2)};
+    &:hover {
+      color: inherit;
+      opacity: 0.64;
+    }
+  `}
+`;
+
+export const MODAL_TYPE = 'modal';
+export const INTERNAL_LINK_TYPE = 'internalLink';
+export const EXTERNAL_LINK_TYPE = 'externalLink';
+export const DROPDOWN_TYPE = 'dropdown';
+
+
+function ChildItem({ child, onClick }: { child: NavLinkItem, onClick?:() => void}) {
```

Comment:
To make the component purpose clearer, how about renaming `ChildItem` to `NavSubItem` or something similar? It might better reflect its role and context in the navigation.

`function NavSubItem({ item, onClick }: { item: NavLinkItem, onClick?:() => void }) {`